05-online-mentoring.md

```diff
@@ -0,0 +1,25 @@
+# Online Mentoring
+
+---
+
+## Tips
+* Prepare like you would for in-person mentoring. Make sure to go over the content
+the mentee is going to be working on.
+* Introduce yourself! Send a message on the slack channel or do a quick video
+meeting introducing yourself to your mentees. This is a great way to become
+familiar and comfortable around each other.
```

Comment:
Love this tip! Could we also make it tech agnostic? I think that would be helpful for other events that might use other software.